code/game/gamemodes/gang/gang.dm

```diff
@@ -193,6 +193,8 @@ var/list/gang_colors_pool = list("red","orange","yellow","green","blue","purple"
 	var/removed
 
 	for(var/datum/gang/G in gangs)
+		if(G.fighting_style == "multiverse" && !remove_bosses)
```

Comment:
Remove it, it is not as if it is going to make a significant difference in a Wizard round.